lib/mli_parser.mli

```diff
@@ -4,9 +4,5 @@ module Code_block : sig
   type t = { location : Odoc_model.Location_.span; contents : string }
 end
 
-val docstring_code_blocks : string -> Code_block.t list
```

Comment:
Do you know why this was exposed?

Reply:
It was exposed since its creating in https://github.com/realworldocaml/mdx/pull/206 but was never used out of this module